src/base.iced

```diff
@@ -29,25 +29,31 @@ exports.Base = class Base
   # Generate an EdDSA keypair from a deterministic seed.
   #
   # @param {Buffer} seed The seed
-  # @return {Object} Contained `publicKey`, `secretKey` buffers
+  # @return {Object} Contained `public_key`, `secret_key` buffers
   # 
   # 
-  genFromSeed : ({seed}) ->
+  gen_from_seed : ({seed}) ->
```

Comment:
yeah, https://github.com/keybase/kbpgp uses it as is.  So you can't snake case things that were previously exposed.